spec/models/task_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  before do
+    FactoryBot.create(:user)
+  end
+  it "【Validation】titleが空ならバリデーションが通らない" do
+    task = FactoryBot.build(:task, title: "")
```

Comment:
設定ファイルを追記することで`FactoryBot.`の部分は省略できます。
```suggestion
    task = build(:task, title: "")
```

Reply:
aaaad08 で修正確認